web_widget_x2many_2d_matrix/static/src/js/2d_matrix_renderer.js

```diff
@@ -51,23 +59,30 @@ odoo.define('web_widget_x2many_2d_matrix.X2Many2dMatrixRenderer', function (requ
          * @returns {Deferred} this deferred is resolved immediately
          */
         _renderView: function () {
+            // ListRenderer._renderView {
```

Comment:
there's no need to keep all these comments. This is not the list renderer nor it inherits from it.
Saying that we were inspired by it is enough.

Reply:
I think it would be beneficial for next generation of contributors to give hints what was replaced or sourced as-is from ListRenderer, for that only purpose

Reply:
I agree with @simahawk. We have git history to know where code comes from, so if you wish, you can add notes in the commit, but don't pollute all of this please. :blush:

Reply:
OK, then just remove the comments please.

Reply:
Removed, although I'm still sure that by doing this we've added extra couple of fun hours for future migrations